### PR TITLE
Add dependency-graph script

### PR DESCRIPTION
**Pull request by benjamincburns**
Generates dependency-graph.pdf in the root of the repo that shows
dependencies between the packages in our workspace. Solid lines reflect
runtime dependencies, dashed lines reflect dev dependencies, and dotted
lines reflect peer dependencies.

### CODE REVIEW

**Comment by benjamincburns** [comment]
Example of the image it generates:

dependency-graph

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "test": "lerna run test --stream --concurrency=1 -- --colors",
     "ci": "./scripts/ci.sh",
     "geth": "./scripts/geth.sh",
+    "dependency-graph": "lerna-dependency-graph -f png -o dependency-graph.png",
```
Comment (gnidan) [inline package.json:12]: Would PDF be better? PNG looks so raster

Comment (benjamincburns) [reply package.json:12]: I'll have a look, but I thought PDF would just encapsulate the raster. If that's the case, I'll also check for a SVG output, as that might be a bit nicer.

Comment (benjamincburns) [reply package.json:12]: Ah, I see what you mean now - PDF is indeed better, as the text is text, so cmd-f works, etc... wilco

**Review by gnidan — APPROVED** [review]
nice to have this easily accessible. thanks!

**Pull request merged**